src/components/Badges/Badges.tsx

```diff
@@ -34,7 +34,7 @@ const guardianPickLabel = css`
 	color: ${neutral[7]};
 `;
 
-export const GuardianStaff = () => (
+export const GuardianStaff = ({ role }: { role: 'Staff' | 'Contributor' }) => (
```

Comment:
You could also have a second component, like  `GuardianContributor` and then there'd be no need for the prop? I think if I were starting from scratch and these were the requirements from day one I would have done that

Fully admit that this comment is a bit opinionated and feely though so whatever feels best to you

Reply:
Actually, this was my initial thought too :) Looking at how the Contributor badge renders via frontend it's identical to the Staff badge, so the GuardianContributor component was a duplication with just the inner text differing. So, I tried to be tidy by giving the component a `role` - might regret that choice soon!

Reply:
Ha, well. One reason for not using `role` is that this name is used _all over_ the place in Guardian code. Around images mainly. Incidentally, the same thing is also called `weight` sometimes but that's just to keep us paying attention.